packages/enhanced/src/lib/container/ContainerEntryDependency.ts

```diff
@@ -46,7 +46,7 @@ class ContainerEntryDependency extends Dependency {
 
 makeSerializable(
   ContainerEntryDependency,
-  'webpack/lib/container/ContainerEntryDependency',
```

Comment:
must be different, webpack automatically requires federation internally, used or not, its serializer gets registered